content/shmem_ctx.tex

```diff
@@ -64,6 +64,13 @@ \subsubsection{\textbf{SHMEM\_CTX\_CREATE}}
 
         \apitablerow{\CONST{SHMEM\_CTX\_PRIVATE}}{
             The given context will be used only by the thread that created it.}
+
+        \apitablerow{\CONST{SHMEM\_CTX\_NOSTORE}}{
+            Quiet and fence operations performed on the given context are not
```

Comment:
This is also a requirement on the application; it cannot rely on quiet/fence on the given context to order store operations.  The option conveys information to the OpenSHMEM library, but it doesn't introduce a new requirement (that is, the library can ignore this option and still be a correct implementation).  Would this be clearer?  "The application does not require quiet and fence operations performed on the given context to complete or order store operations."